.travis.yml

```diff
@@ -26,24 +26,28 @@ env:
 matrix:
   include:
    - php: 5.2
-     env: WP_VERSION=latest
+     env: WP_VERSION=latest WC_VERSION=2.6.14
    - php: 5.2
-     env: WP_VERSION=4.4
+     env: WP_VERSION=4.4 WC_VERSION=2.6.14
    - php: 5.3
-     env: WP_VERSION=latest
+     env: WP_VERSION=latest WC_VERSION=2.6.14
    - php: 5.3
-     env: WP_VERSION=4.4
+     env: WP_VERSION=4.4 WC_VERSION=2.6.14
    - php: 5.4
-     env: WP_VERSION=latest
+     env: WP_VERSION=latest WC_VERSION=2.6.14
    - php: 5.4
-     env: WP_VERSION=4.4
+     env: WP_VERSION=4.4 WC_VERSION=2.6.14
    - php: 5.5
-     env: WP_VERSION=latest
+     env: WP_VERSION=latest WC_VERSION=2.6.14
    - php: 5.5
-     env: WP_VERSION=4.4
+     env: WP_VERSION=4.4 WC_VERSION=2.6.14
+   - php: 5.5
+     env: WP_VERSION=latest WC_VERSION=3.0.3
      # 5.6 / master already included above as first build.
    - php: 5.6
-     env: WP_VERSION=4.4
+     env: WP_VERSION=4.4 WC_VERSION=2.6.14
+   - php: 5.6
+     env: WP_VERSION=4.4 WC_VERSION=3.0.3
```

Comment:
There's an ~app~ issue for that: https://github.com/Automattic/woocommerce-services/issues/780

We can discuss in that issue the combinations (WC/WP/PHP) we should need to test (more than 10 would seem overkill for me, specially if we aren't supporting WordPress all the way back to 4.4).

Let's not block this issue any longer :)